pycm/pycm_obj.py

```diff
@@ -218,6 +218,26 @@ def __iter__(self):
         for key in self.matrix.keys():
             yield key, self.matrix[key]
 
+    def __contains__(self, class_name):
+        """
+        Check if the confusion matrix contains given class name.
+
+        :param class_name: given class name
+        :param class_name: any valid type
+        :return: True if class_name is a class in confusion matrix
+        """
+        return class_name in self.classes
+
+    def __getitem__(self, class_name):
+        """
+        Return value in matrix corresponding to the given class name.
```

Comment:
"Return the element(s) in the matrix corresponding to the given class name."

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/476/commits/b8e1dc4c6168621d549dee146847de11011a8fcd